style.css

```diff
@@ -52,6 +55,65 @@ header {
   transform: scale(1.7);
 }
 
+nav ul {
+  display: flex;
+  list-style: none;
+}
+
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: #1d2fd5;
+  mix-blend-mode: multiply;
+  transition: 200ms ease-in-out;
+  transform: scale(0);
+  z-index: 10;
+}
+
+.mobile-menu.active {
+  transform: scale(1);
+}
+
+.mobile-menu-header {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.mobile-menu ul {
+  list-style: none;
+  color: #fff;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.mobile-menu-list a {
+  color: white;
+  text-decoration: none;
+}
```

Comment:
- [ ] [ optional] It would be very good if the background of the navbar on the mobile menu is not transparent, from that you can make user experience much better

https://user-images.githubusercontent.com/55918152/140894356-c362a3f8-16d0-4c59-ac0b-f52d02d81c52.mp4

Reply:
Resolved